internal/querynode/impl.go

```diff
@@ -385,24 +385,42 @@ func (node *QueryNode) ReleaseSegments(ctx context.Context, in *queryPb.ReleaseS
 		}
 		return status, err
 	}
-	status := &commonpb.Status{
-		ErrorCode: commonpb.ErrorCode_Success,
+	dct := &releaseSegmentsTask{
```

Comment:
if the segment to be released not exist in historical replica, the there is not need to create a task.

Reply:
fixed